code/datums/elements/turf_transparency.dm

```diff
@@ -56,20 +53,20 @@
 	if(dir != DOWN)
 		return
 
-	update_multiz(our_turf)
+	update_multi_z(our_turf)
 
 /datum/element/turf_z_transparency/proc/on_multiz_turf_new(turf/our_turf, turf/below_turf, dir)
 	SIGNAL_HANDLER
 
 	if(dir != DOWN)
 		return
 
-	update_multiz(our_turf)
+	update_multi_z(our_turf)
 
 ///Called when there is no real turf below this turf
-/datum/element/turf_z_transparency/proc/show_bottom_level(turf/our_turf)
-	if(!show_bottom_level)
-		return FALSE
+/datum/element/turf_z_transparency/proc/add_baseturf_underlay(turf/our_turf)
+	if(istype(our_turf, /turf/open/openspace)) // we don't ever want our bottom turf to be openspace
```

Comment:
Also do not like this, but I think I get what you're going for

Reply:
We don't always want to reset to base right? or was the old intent of that check wrong

Reply:
I had to rename the proc because it was misleading and conflicting with another variable that had the same name.

If the bottom z-level is openspace then a mapper made an opsie.  This fixes any openspace tiles on the bottom z-level to be changed to plating as a safety precaution.  I should probably throw in some logging since if this is happening either an admin or a mapper did something wrong since it shouldn't be happening.

Reply:
I meant the typecheck, it seems fragile to assume that typepath is the only unacceptable bottom

Reply:
See my above comment about openspace behaviour resembling bedrock in minecraft.